<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdCom</class>
 <widget class="QMainWindow" name="WdCom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QSpinBox" name="spNum"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Тип РЛС</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Номер РЛС</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbType">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Сценарий</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbCom">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Сектор ответственности</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Номер команды</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="pbSave">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Состояние</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Тип команды</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbSector">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Темп обзора</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbNumber">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="cbScena">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cbSost">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="dsbTemp"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
